jaseci_core/jaseci/api/alias_api.py

```diff
@@ -23,7 +23,7 @@ class alias_api():
     def __init__(self):
         self.alias_map = {}
 
-    @interface.private_api()
+    @interface.private_api(cli_args=['name'])
```

Comment:
So this means that `alias register` will have the default arguments `name`, correct? 

```
alias register NAME
```

Reply:
Correct! No need for `alias register -name NAME -value VALUE` you can just do `alias register NAME -value VALUE`